src/robusta/core/playbooks/common.py

```diff
@@ -1,34 +1,74 @@
-from typing import Optional
+from typing import Optional, List
 from hikaru.model import EventList, Event
 
 from ..reporting import TableBlock
 from ..reporting.custom_rendering import RendererType
 from ...integrations.kubernetes.api_client_utils import parse_kubernetes_datetime_to_ms
 
 
-def get_resource_events_table(table_name: str, kind: str, name: str, namespace: str = None) -> Optional[TableBlock]:
-    field_selector = f"involvedObject.kind={kind},involvedObject.name={name}"
+def filter_event(
+    ev: Event, name_substring_filter: str, included_types: Optional[List[str]]
+) -> bool:
+    if name_substring_filter not in ev.involvedObject.name:
```

Comment:
I didn't see any such usage, but if this is called with None, what would be the result?
Should we be extra careful and handle it?

Reply:
Sure.

Reply:
fixed